.github/workflows/ci.yml

```diff
@@ -39,13 +39,24 @@ jobs:
       run: ./gradlew clean build -Dbuild.snapshot=false -x test
 
     - name: Test
-      run: OPENDISTRO_SECURITY_TEST_OPENSSL_OPT=true ./gradlew test -i
```

Comment:
No need to diable `info` logs in CI logs.

Reply:
All the output is still available in the logs if you download the bundle.  My browser chugs when I load the CI build output so this was also to make that part of the GHA workflow better to use.

Reply:
View raw logs will avoid downloading the bundle. And it can be rendered in brower without issues. Let's keep the info logs available to check.
![Screen Shot 2022-04-21 at 4 04 39 PM](https://user-images.githubusercontent.com/16925774/164565294-09b61588-2cec-42bb-8305-d374861f63a8.png)